Packs/Akamai_SIEM/ReleaseNotes/1_1_6.md

```diff
@@ -0,0 +1,9 @@
+
+#### Integrations
+
+##### Akamai WAF SIEM
+
+- Added the **Page size** integration parameter. Use this parameter to determine how many events to fetch on each request being made to Akamai.
+- Fixed an issue where the integration sometimes returned duplicated events.
+- Updated the Docker image to: *demisto/auth-utils:1.0.0.115527*.
+- Added the **akamai-siem-reset-offset** command which allows to reset the integration context. Use this command when you want to remove offset history when fetching events.
```

Comment:
Added the **akamai-siem-reset-offset** command which allows you to reset the integration context. Use this command when you want to remove offset history when fetching events.